packages/api/endpoints/granules.js

```diff
@@ -140,37 +141,50 @@ async function del(req, res) {
   log.info(`granules.del ${granuleId}`);
 
   const granuleModelClient = new models.Granule();
+  const granulePgModel = new GranulePgModel();
+  const filePgModel = new FilePgModel();
 
-  let granule;
+  const knex = await getKnexClient({ env: process.env });
+
+  let dynamoGranule;
+  let pgGranule;
   try {
-    granule = await granuleModelClient.getRecord({ granuleId });
+    dynamoGranule = await granuleModelClient.getRecord({ granuleId });
+    pgGranule = await granulePgModel.get(knex, { granule_id: granuleId });
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
       return res.boom.notFound(error);
     }
     throw error;
   }
 
-  if (granule.detail) {
-    return res.boom.badRequest(granule);
+  // TODO what is this?
+  if (dynamoGranule.detail) {
+    return res.boom.badRequest(dynamoGranule);
   }
```

Comment:
I'm not sure exactly what we're checking for here. This doesn't look like its in our original Dynamo schema. Do we still need this?

Reply:
I think this is some leftover code that was never removed. It seems like it was originally reading a `detail` field from the granule schema:

https://github.com/nasa/cumulus/commit/a7a4331380c718bf101d0035cb1fa2744ff9fc79

But as you point out, that field no longer exists. Ultimately, we should just remove this code and see if any tests start failing. But I don't know if I would include that in this PR just to keep things contained

Reply:
Sounds good. For now I'm going to leave a "DEPRECATED" comment and note that it can be removed. I'll circle back after everything else here is taken care of and see if it causes anything to fail.